mappings/net/minecraft/entity/ai/goal/TrackTargetGoal.mapping

```diff
@@ -9,6 +9,7 @@ CLASS aom net/minecraft/entity/ai/goal/TrackTargetGoal
 	FIELD h maxTimeWithoutVisibility I
 	METHOD <init> (Laij;ZZ)V
 		ARG 2 checkVisibility
+		ARG 3 checkCanNavigate
```

Comment:
`checkNavigable`

Reply:
Everywhere else in this function uses `canNavigate`, is it fine make this different?